libs/ng-utils/src/lib/use-http-image-source/use-http-image-source.pipe.ts

```diff
@@ -39,6 +45,27 @@ export class UseHttpImageSourcePipe implements PipeTransform, OnDestroy {
     this.setUpSubscription();
   }
 
+  /**
+   * Returns a valid url path (blob path, loadingImagePath or errorImagePath) or null based on image fetching state.
+   *
+   * @example
+   * // Inject UseHttpImageSourcePipe into module
```

Comment:
Well, that is not how we "inject" (in reality we import the ngModules into modules to reuse them) thing. And this TsDoc entry should document the transform method.

Technically it gets only called when you use it in a template, and users don't usually call `.transform()` manually.
I'd say remove the documentation from the transform method and add a class documentation.

The class documentation is basically the example from the lib readme file